dashboard.js

```diff
@@ -1,10 +1,10 @@
 const cookieExist=document.cookie;
 const cookieDeatails=cookieExist.split("=")
 if (cookieDeatails[1]===undefined){
-    window.location.href = "http://127.0.0.1:5500/index.html";
+    window.location.href = "https://aiswarya-a-m.github.io/login/";
 }
 document.querySelector("#nameOfUser").innerHTML=cookieDeatails[1];
 function deleteItem(){
```

Comment:
Give meaningful names and name that denotes its functionality. What item we are supposed to delete in this function?